core/src/main/scala/org/apache/spark/rdd/ParallelCollectionRDD.scala

```diff
@@ -32,8 +32,8 @@ import org.apache.spark.util.Utils
 private[spark] class ParallelCollectionPartition[T: ClassTag](
     var rddId: Long,
     var slice: Int,
-    var values: Seq[T])
-    extends Partition with Serializable {
+    var values: Seq[T]
+  ) extends Partition with Serializable {
```

Comment:
isn't `extends Partition with Serializable {` should be 2-indent instead of 4-indent?
I see many of similar style where 2-indent is used https://github.com/apache/spark/blob/master/streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala#L63